pkg/csi/service/wcp/controller.go

```diff
@@ -893,3 +893,11 @@ func (c *controller) ControllerExpandVolume(ctx context.Context, req *csi.Contro
 	}
 	return resp, err
 }
+
+func (c *controller) ControllerGetVolume(ctx context.Context, req *csi.ControllerGetVolumeRequest) (
+	*csi.ControllerGetVolumeResponse, error) {
+	ctx = logger.NewContextWithLogger(ctx)
```

Comment:
we do not need to initialize context and logger for Unimplemented interfaces.
we just need to return `return nil, status.Error(codes.Unimplemented, "")`